functions/fttx_consume/deploy.sh

```diff
@@ -0,0 +1,7 @@
+gcloud functions deploy vwt-d-gew1-fttx-dashboard-consume-func \
```

Comment:
@agvanturnhout use substituting variables to define project instead of committing the full name.

Reply:
The deploy.sh file is only used for deploying the function from the command line when testing. This is faster than triggering the whole build. Therefore only vwt-d-gew1-fttx-dashboard-consume-func is used.